addons/hr_recruitment/models/hr_job.py

```diff
@@ -246,10 +252,15 @@ def _alias_get_creation_values(self):
             })
         return values
 
+    @api.onchange('name')
+    def _compute_alias_name(self):
+        for job in self:
+            job.alias_name = self.env['ir.http']._slugify_one(job.name or "")
```

Comment:
Please remove this, we don't want to create automatic aliases or even worse aliases with empty names.